src/resources/elements/numericInput/numericInput.ts

```diff
@@ -14,6 +14,7 @@ export class NumericInput {
   @bindable.booleanAttr public decimal = true;
   @bindable public css?: string;
   @bindable({ defaultBindingMode: bindingMode.oneTime }) public id?: string;
+  @bindable(({ defaultBindingMode: bindingMode.twoWay })) inFocus = false;
```

Comment:
why is inFocus needed?

Reply:
I added this just to make the input focus when you use the "inFocus" attribute on the other "pinput-numeric" component. The `pinput-numeric` has a specific border color when focus. And when it's in focus you should also have the focus in the input itself. The "focus" border color alone didn't make sense without having an input which is not in focus.